worker/src/DepLibSRTP.cpp

```diff
@@ -45,17 +49,31 @@ void DepLibSRTP::ClassInit()
 {
 	MS_TRACE();
 
-	MS_DEBUG_TAG(info, "libsrtp version: \"%s\"", srtp_get_version_string());
+	{
+		std::lock_guard<std::mutex> lock(globalSyncMutex);
+		if (globalInstances == 0)
+		{
+			MS_DEBUG_TAG(info, "libsrtp version: \"%s\"", srtp_get_version_string());
 
-	srtp_err_status_t err = srtp_init();
+			srtp_err_status_t err = srtp_init();
 
-	if (DepLibSRTP::IsError(err))
-		MS_THROW_ERROR("srtp_init() failed: %s", DepLibSRTP::GetErrorString(err));
+			if (DepLibSRTP::IsError(err))
+				MS_THROW_ERROR("srtp_init() failed: %s", DepLibSRTP::GetErrorString(err));
+		}
+		++globalInstances;
```

Comment:
Please blank line after code clocks everywhere:

```suggestion
		}

		++globalInstances;
```

Also, is the purpose of those `globalSyncMutex` just protect the logging based on `globalInstances`?

Reply:
That and `srtp_init()` call, since it only needs to be called once globally.

Linter didn't complain about that :shrug:
Fixed :slightly_smiling_face: